webforj-plugins/webforj-install-bbj-plugin/src/main/resources/log4j2.xml

```diff
@@ -20,7 +20,7 @@
     >
       <PatternLayout pattern="${LOG_PATTERN}"/>
       <Policies>
-        <SizeBasedTriggeringPolicy size="500KB"/>
+        <SizeBasedTriggeringPolicy size="1024KB"/>
```

Comment:
Why this change ?

Reply:
???

Reply:
done.  It is a preference, I rolled it back, will make argument for it again in the future